src/ipyaladin/widget.py

```diff
@@ -1020,6 +1068,8 @@ def set_listener(self, listener_type: str, callback: Callable) -> None:
             self.listener_callback["click"] = callback
         elif listener_type == "select":
             self.listener_callback["select"] = callback
+        elif listener_type == "current_overlays":
+            self.listener_callback["current_overlays"] = callback
         else:
             raise ValueError(
                 "listener_type must be 'object_hovered', "
```

Comment:
we need to update this error message to also include `current_overlays` now that we've added this as a listener